site/dat/docs/Installation.md

```diff
@@ -147,4 +148,4 @@ UNSTABLE_SNAPSHOT
 
 ## Data Collection Disclaimer
```

Comment:
I don't know why the last line is showing a difference. I'm guessing there was a trivial line-ending change.